protocols/imap/src/main/java/org/apache/james/imap/processor/AbstractAuthProcessor.java

```diff
@@ -129,9 +130,11 @@ private void provisionInbox(ImapSession session, MailboxManager mailboxManager,
         } else {
             try {
                 Optional<MailboxId> mailboxId = mailboxManager.createMailbox(inboxPath, mailboxSession);
-                LOGGER.info("Provisioning INBOX. {} created.", mailboxId);
+                LOGGER.info("Provisioning INBOX. {} created.", mailboxId.get());
```

Comment:
what about handling the absence with a if rather than throwing and catching 2 lines after ?

Reply:
I'm a bit reluctant to add a if, at least it's treated as another error case

Reply:
fixed with @chibenwa suggestion